atomic_reactor/plugins/pre_check_and_set_rebuild.py

```diff
@@ -94,15 +98,17 @@ def run(self):
                                       namespace=metadata.get('namespace', None))
             osbs = OSBS(osbs_conf, osbs_conf)
             labels = {self.label_key: self.label_value}
-            try:
-                osbs.set_labels_on_build_config(buildconfig, labels)
-            except OsbsResponseException as ex:
-                if ex.status_code == 409:
-                    # Someone else was modifying the build
-                    # configuration at the same time. Try again.
-                    self.log.debug("got status %d, retrying", ex.status_code)
+
+            for counter in xrange(times):
+                if counter > 0:
+                    sleep(delay)
+                try:
                     osbs.set_labels_on_build_config(buildconfig, labels)
```

Comment:
This already performs retries for 409 conflict: https://github.com/projectatomic/osbs-client/blob/master/osbs/core.py#L688